starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py

```diff
@@ -228,17 +220,17 @@ def verify(self, request, pk=None):
         item = self.get_object()
 
         connection_response = requests.get(
-            f"{AGENT_ADMIN_URL}/connections?alias={django.conf.settings.AGENT_SELF_CONNECTION_ALIAS}",
-            headers=ADMIN_REQUEST_HEADERS,
+            f"{django.conf.settings.AGENT_ADMIN_URL}/connections?alias={django.conf.settings.AGENT_SELF_CONNECTION_ALIAS}",
```

Comment:
Not sure why the top-level import id for the full `django` package, we should instead be doing `from django.conf import settings` to have more granular control on imports. It looks like this has been there for a bit, if some other django sub-package is being used we should probably explicitly declare it rather than bulk-import django.